src/main/java/mate/controller/driver/AddDriverController.java

```diff
@@ -24,8 +24,8 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
     protected void doPost(HttpServletRequest req, HttpServletResponse resp)
             throws IOException, ServletException {
         String name = req.getParameter("name");
-        String licenceNumber = req.getParameter("licence_number");
-        Driver driver = new Driver(name, licenceNumber);
+        String licenseNumber = req.getParameter("licence_number");
```

Comment:
```suggestion
        String licenseNumber = req.getParameter("license_number");
```
І відповідно в усіх скриптах до і після цього пул реквесту

Reply:
Це єдиний репо в якому є готові контролери та JSP